scripts/tap-generate-docs/test/tap_generate_docs_test.clj

```diff
@@ -279,13 +288,11 @@
        "type" "array",
        "description" ""})))
 
-;; These are not throwing anymore, since an empty properties schema is valid. They now log.
-;;; TODO Change this to a positive assertion
-#_(testing "Null types"
-    (is (thrown? clojure.lang.ExceptionInfo
-                 (convert-multiary-type nil ["a_null" {"type" "null"}])))
-    (is (thrown? clojure.lang.ExceptionInfo
-                 (convert-multiary-type nil ["a_null" {"type" ["null"]}]))))
+(testing "Null types"
+    (is (not (thrown? clojure.lang.ExceptionInfo
+                  (convert-multiary-type nil ["a_null" {"type" "null"}]))))
+    (is (not (thrown? clojure.lang.ExceptionInfo
+                  (convert-multiary-type nil ["a_null" {"type" ["null"]}])))))
```

Comment:
I think it would be better for us to test the actual return value these should be now (`nil?`, a `null` type doc partial?), rather than that they simply don't throw.